src/browser/api/channel.ts

```diff
@@ -110,8 +109,32 @@ export module Channel {
         return providerIdentity;
     }
 
+    export function destroyChannel(identity: Identity, channelName: string): void {
```

Comment:
Is this always called from within a `try` or `.catch` code path?

Reply:
Yep, only codepath is normal API call